tlspuffin/src/tls/fn_extensions.rs

```diff
@@ -636,41 +642,41 @@ pub fn fn_transport_parameters_draft_server_extension(
 pub fn fn_unknown_client_extension() -> Result<ClientExtension, FnError> {
     Ok(ClientExtension::Unknown(UnknownExtension {
         typ: ExtensionType::Unknown(0xFFFF),
-        payload: Payload::new([42; 7000]),
+        payload: Payload::new([42; 10]),
     }))
 }
 
 pub fn fn_unknown_server_extension() -> Result<ServerExtension, FnError> {
     Ok(ServerExtension::Unknown(UnknownExtension {
         typ: ExtensionType::Unknown(0xFFFF),
-        payload: Payload::new([42; 7000]),
+        payload: Payload::new([42; 10]),
```

Comment:
Are there any reason to use 10 as payload size and why was it 7000 before ? I suppose that 7000 was impacting the amount of bitlevel modifications that could happen is such a large payload.